src/lib/features/events/event-store.ts

```diff
@@ -380,6 +380,32 @@ class EventStore implements IEventStore {
         return query;
     }
 
+    async getEventCreators(): Promise<Array<{ id: number; name: string }>> {
```

Comment:
1. -1337 and -42 get their name automatically from our predefined constants
2. Then looks at users table
3. Finally looks at events table